modules/apps/object/object-test/src/testFunctional/paths/ObjectNotifications.path

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<title>ObjectNotifications</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ObjectNotifications</td></tr>
+</thead>
+
+<tbody>
+	<tr>
+		<td>DATA_SOURCE_ATTACHMENTS</td>
+		<td>//div[@tabindex='0']//span[contains(.,'${key_dataSource}')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>DATA_SOURCE_ATTACHMENTS_SELECT_OBJECT</td>
+		<td>//div[@class='dropdown-menu show']//div[@class='d-flex justify-content-between' and contains(.,'Custom Object 161924')]</td>
+		<td></td>
+	</tr>
```

Comment:
We shouldn't hardcode the Object name here since we can't use the path in other tests